package.json

```diff
@@ -40,7 +40,7 @@
     "@types/offscreencanvas": "^2019.6.2",
     "@types/serve-index": "^1.9.1",
     "@typescript-eslint/parser": "^4.22.0",
-    "@webgpu/types": "^0.1.7",
+    "@webgpu/types": "^0.1.8",
```

Comment:
Will need rebase to remove the carat `^`.

I just released 0.1.9 so you should be able to update now!